docs/build/building-modules/19-module-account.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 1
+---
+
+# Module Accounts
+
+## Address Generation
```

Comment:
That's a bit short for it to be in a dedicated page maybe. Having it in building modules is as well weird. Maybe an advanced concept?